pkg/client/cli/cliutil/exec_windows.go

```
@@ -0,0 +1,44 @@
+package cliutil
+
+import (
+	"context"
+	"os"
+	"os/signal"
+
+	"golang.org/x/sys/windows"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/logging"
+	"github.com/telepresenceio/telepresence/v2/pkg/proc"
+)
+
+func background(ctx context.Context, exe string, args []string) error {
+	return shellExec(ctx, "open", exe, args)
+}
+
+func backgroundAsRoot(ctx context.Context, exe string, args []string) error {
+	verb := "runas"
+	if proc.IsAdmin() {
+		verb = "open"
+	}
+	return shellExec(ctx, verb, exe, args)
+}
+
+func shellExec(_ context.Context, verb, exe string, args []string) error {
+	cwd, _ := os.Getwd()
+	verbPtr, _ := windows.UTF16PtrFromString(verb)
+	exePtr, _ := windows.UTF16PtrFromString(exe)
+	cwdPtr, _ := windows.UTF16PtrFromString(cwd)
```

It looks like these can only error if the string contains a NUL byte somewhere, so it makes sense to ignore the error -- should we add a short comment explaining this?